doc/nrf/device_guides/working_with_nrf/nrf52/developing.rst

```diff
@@ -90,14 +129,16 @@ To perform a FOTA update, complete the following steps:
 
    |fota_upgrades_building|
 
-#. Download the :file:`app_update.bin` image file to your device.
+#. Download the :file:`dfu_application.zip` zip archive or the :file:`app_update.bin` image file to your device.
+   What file should be chosen depends on the application used to perform the firmware update.
+   See :ref:`app_build_fota` for more information about update files.
```

Comment:
I think we should also modify the `app_build_fota` documentation section to cover the generated files in case of the Direct XIP mode (you can also add a note that the direct-xip is only supported on the single core devices - nRF52 series)

Reply:
I have noted in the build output files array that Direct-xip output is for single-core devices.